desktop/app.js

```diff
@@ -30,8 +30,13 @@ module.exports = function main() {
   app.on('will-finish-launching', function () {
     setTimeout(updater.ping.bind(updater), config.updater.delay);
     app.on('open-url', function (event, url) {
+      let deeplinkingUrl = url;
       event.preventDefault();
-      mainWindow.webContents.send('wpLogin', url);
+      if (process.platform === 'win32') {
+        // Keep only command line / deep linked arguments
+        deeplinkingUrl = process.argv.slice(1);
+      }
```

Comment:
no need for the mutation: can easily assign this as a `const`

```ts
const deeplinkingUrl = process.platform === 'win32'
	? process.argv.slice(1) // keep only command-line / deep-linked arguments
	: url;
```

also, just a check - we want _either_ the given URL _or_ an array of strings for this value?